client/settings/save-settings-section/index.js

```diff
@@ -56,6 +77,32 @@ const SaveSettingsSection = ( { disabled = false } ) => {
 				settings.is_payment_request_enabled
 			);
 		}
+
+		// Show the feedback modal when WooPay is disabled.
+		if ( initialIsWooPayEnabled && ! settings.is_woopay_enabled ) {
+			const { woopayLastDisableDate } = wcpaySettings;
+
+			// Do not show feedback modal if WooPay
+			// was disabled in the last 7 days.
+			if ( woopayLastDisableDate ) {
+				const date1 = new Date( woopayLastDisableDate );
+				const date2 = new Date();
+				const diffTime = Math.abs( date2 - date1 );
+				const diffDays = Math.ceil(
+					diffTime / ( 1000 * 60 * 60 * 24 )
+				);
+
+				if ( diffDays < 7 ) {
+					return;
+				}
+			}
+
+			setIsWooPayDisableFeedbackOpen( true );
+
+			// Prevent show modal again.
+			setInitialIsPaymentRequestEnabled( true );
+			wcpaySettings.woopayLastDisableDate = new Date();
```

Comment:
```suggestion
            // Set last disable date to prevent feedback window opening up on successive "Save button" clicks. This value is overwritten on page refresh.
			wcpaySettings.woopayLastDisableDate = new Date();
```